src/io/flutter/view/InspectorTreeUI.java

```diff
@@ -67,8 +68,6 @@
 
   boolean leftToRight = true; /// TODO(jacobr): actually support RTL mode.
 
-  static final JBColor SUBTREE_BOUNDS_COLOR = new JBColor(Color.WHITE, Gray._43);
```

Comment:
The only thing that actually changed in this file is that SUBTREE_BOUNDS_COLOR was removed 
and 
the custom paint method in this class was removed.